webpack/reducers/forms_reducer.js

```diff
@@ -0,0 +1,12 @@
+import {
+  FETCH_FORMS_FULFILLED
+} from '../actions/types';
+
+const initialState = null;
+
+export default function forms(state = initialState, action) {
+  if (action.type == FETCH_FORMS_FULFILLED) {
+    return [...action.payload.data];
```

Comment:
My bad, I modified this in the course of chasing down that 500 error, I'll make sure this is changed -- and that the attendant changes are made elsewhere as well.